rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -580,7 +580,7 @@
     REGISTER_BTC_TRANSFER(
             CallTransaction.Function.fromSignature(
                     "registerBtcTransfer",
-                    new String[]{"bytes", "uint256", "bytes", "bytes32", "string", "string", "string", "bool"},
+                    new String[]{"bytes", "uint256", "bytes", "bytes32", "bytes", "string", "bytes", "bool"},
```

Comment:
Not sure if it makes a difference, but you could use `bytes20` as the data type for btc addresses since they are always 20 bytes long